tests/certs/generate_certs

```diff
@@ -0,0 +1,7 @@
+openssl req -x509 -newkey rsa:4096 -nodes -keyout ca.key -out ca.pem -sha256 -days 3650 -subj "/C=US/ST=New York/O=openziti.org/CN=test CA"
```

Comment:
is this a markdown?  a script? why bother checking them in if it's easy enough to regenerate them? meh... i'd add some kind of extension just as a "fyi" to others